### ISSUE TITLE
Lootr chests lose their translated name once opened

### ISSUE THREAD

**Issue by Delfayne** [FTBTeam/FTB-Modpack-Issues#5673]
### Modpack

FTB Interactions Remastered

### Modpack version

1.01

### Has the pack been modified

No

### Describe the bug

When opening Lootr chests, the name changes to the translation key rather than the translated value after being opened.

### Steps to reproduce

Open a Lootr chest and observe the reported name change in TOP

### Expected behaviour

To maintain their translated name, or correctly use a new translated name

### Screenshots

<img width="414" alt="image" src="https://github.com/user-attachments/assets/24ce5027-b54a-48be-8fae-fece87088f00">

### Additional information

This occurs in multiplayer. I have not tested single player.

**Comment by ArtpokeBr** [comment]
Yeah.. this might be an issue with the Lootr Mod, please make sure to report to them.
